docs/content/adoption/companies_experience.md

```diff
@@ -0,0 +1,32 @@
+# Companies experience
+
+Looking to how other companies implemented `UI/Snapshot testing` may help you to choose proper tools for your project
+and save your time.
+
+We would be really grateful if you
+could [contribute and share your experience](https://android-ui-testing.github.io/Cookbook/home/contribution) page to
+help other people.
+
+## Revolut
+
+`UI testing`
+
+:   
+`Write:` [Kaspresso](https://github.com/KasperskyLab/Kaspresso) <br>
+`Who:` Android Engineers<br>
+`Runner:` [Marathon](https://android-ui-testing.github.io/Cookbook/practices/test_runners_review/#21-marathon) <br>
+`Where:` [Headless emulators in Docker (Avito Image)](https://hub.docker.com/r/avitotech/android-emulator-29) <br>
```

Comment:
I guess quite important also hardware specification (but i don't know is it NDA or not)